web/app/test/ServiceMeshTest.jsx

```diff
@@ -0,0 +1,79 @@
+import ServiceMesh from '../js/components/ServiceMesh.jsx';
+import { expect } from 'chai';
+import { mount, shallow } from 'enzyme';
+import podFixtures from './fixtures/pods.json';
+import React from 'react';
+import sinon from 'sinon';
+import _ from 'lodash';
+import sinonStubPromise from 'sinon-stub-promise';
+import { printStack, routerWrap } from './testHelpers.jsx';
```

Comment:
we don't use printStack so I'd remove that import

Reply:
got it!